mindmeld/models/text_models.py

```diff
@@ -386,7 +386,7 @@ def _get_feature_scaler(self):
         }.get(scale_type)
         return scaler
 
-    def evaluate(self, examples, labels):
+    def evaluate(self, examples, labels, active_learning=False):
```

Comment:
Let's rename this parameter to focus on the updated return type instead of a single application it could be used for. This could be "ret_prob_full" or "ret_prob_all" or anything else you see fit. In this way, this function can easily/intuitively be used for applications beyond AL.